samples/games/kotlin/build.gradle.kts

```diff
@@ -51,14 +53,23 @@ kotlin {
         }
     }
 
-    val targets = listOf(
-            targetFromPreset(presets["godotMingwX64"], "windows"),
-            targetFromPreset(presets["godotLinuxX64"], "linux"),
-            targetFromPreset(presets["godotMacosX64"], "macos")
-    )
+    val targets = if (project.hasProperty("platform")) {
+        when (platform) {
+            "windows" -> listOf(targetFromPreset(presets["godotMingwX64"], "windows"))
+            "linux" -> listOf(targetFromPreset(presets["godotLinuxX64"], "linux"))
+            "macos" -> listOf(targetFromPreset(presets["godotMacosX64"], "macos"))
+            else -> listOf(targetFromPreset(presets["godotMacosX64"], "macos"))
+        }
+    } else {
+        listOf(
+                targetFromPreset(presets["godotLinuxX64"], "linux"),
+                targetFromPreset(presets["godotMacosX64"], "macos"),
+                targetFromPreset(presets["godotMingwX64"], "windows")
+        )
+    }
 
-    targets.forEach { target ->
-        target.compilations.getByName("main") {
+    targets.forEach {
```

Comment:
Isn't dropping the named lambda argument a step in the wrong direction?

Reply:
I think not as `it.compilations.getByName("main")` also has a `target` so in it's block it's hard to understand which `target` we mean.
But i pushed a change which should clarify this by adding an explicit `this` before the usages. Feel free to look at that and say what you think about it :-)

Reply:
@lassem Ok for you if i leave it like this?